module.json

```diff
@@ -27,6 +27,9 @@
     "nanostack": {
       "mbed-mesh-api": "~0.0.1"
     },
+    "/mbed-os/net/stacks/nanostack": {
```

Comment:
If you do this, shouldn't you also delete the always-on dependency on `nanostack` above ?